microservices/realtime/install

```diff
@@ -0,0 +1,16 @@
+$ sudo pecl install swoole-4.3.5
```

Comment:
This is now available as php-swoole package in extra component of our repository